### PR TITLE
Enforcement form

### PR DESCRIPTION
**Pull request by k-gregory**

### CODE REVIEW

File: backend/src/main/resources/application.yml [hunk 1]
```diff
@@ -57,4 +57,4 @@ spring:
   flyway.locations: classpath:db/migration/h2
 logging.level:
   org.hibernate.SQL: DEBUG
-  org.hibernate.type.descriptor.sql.BasicBinder: TRACE
```
Comment (kostya9) [inline backend/src/main/resources/application.yml:60]: is this intended?

File: frontend/src/api/enforcement.ts [hunk 2]
```diff
@@ -20,3 +21,7 @@ export const fetchTopEnforcements = () => axios
     .get('/api/enforcement/top')
     .then((response) => response.data.map(mapEnforcement));
 
+export const postEnforcement = (data: Enforcement) => axios
```
Comment (kostya9) [inline frontend/src/api/enforcement.ts:24]: No issue with this, just wondering - why arrow functions?

File: backend/src/main/resources/application.yml (hunk 1 above)
Comment (k-gregory) [reply backend/src/main/resources/application.yml:60]: No, seems like it was deleted by IDEA unused code remover

File: frontend/src/api/enforcement.ts (hunk 2 above)
Comment (kostya9) [reply frontend/src/api/enforcement.ts:24]: Let's explicitly show function return type

File: backend/src/main/java/io/github/k_gregory/registry/controller/EnforcementController.java [hunk 3]
```diff
@@ -1,25 +1,85 @@
...
 
     @RequestMapping("/top")
     public List<TopEnforcementDTO> topEnforcements() {
         return enforcement.listTopEnforcements();
     }
+
+    @PostMapping("/add")
+    @ResponseBody
+    @Transactional
+    public String addEnforcement(@RequestBody EnforcementDTO body) {
```
Comment (kostya9) [inline backend/src/main/java/io/github/k_gregory/registry/controller/EnforcementController.java:69]: Let's move this logic to service

**Pull request closed without merging**